base.gradle

```diff
@@ -71,8 +70,10 @@ dependencies {
 
     debugImplementation libs.compose.ui.test.manifest
     debugImplementation libs.compose.ui.tooling
-    debugProdImplementation libs.compose.ui.test.manifest
-    debugProdImplementation libs.compose.ui.tooling
+    if (configurations.findByName("debugProdImplementation") != null) {
+        debugProdImplementation libs.compose.ui.test.manifest
+        debugProdImplementation libs.compose.ui.tooling
+    }
```

Comment:
I don't know if these dependencies are needed in application modules. I kept them just in case.